src/includes/getting-started-config/native.qt.mdx

```diff
@@ -0,0 +1,34 @@
+Import and initialize the Sentry SDK early in your application setup:
```

Comment:
No need for this as the common content already has this info.

Reply:
Ah, I copied native.mdx, which has the same issue, I'll fix it there.